src/views/GameView.vue

```diff
@@ -130,12 +126,7 @@
       <!-- Draw / Scrap Piles -->
       <div class="deck-container">
         <div id="field-left">
-          <v-card
-            id="deck"
-            :img="require('@/img/cards/Bg_Deck.png')"
-            :class="{ 'reveal-top-two': resolvingSeven }"
-            @click="drawCard"
-          >
+          <v-card id="deck" :class="{ 'reveal-top-two': resolvingSeven }" @click="drawCard" elevation="0">
```

Comment:
Todo: if we don't already have an issue for it, write one to fix the shitty looking "choose from dreck" text with its poor contrast on the card back img

Reply:
https://github.com/cuttle-cards/cuttle/issues/170